examples/vue/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="icon" type="image/svg+xml" href="./favicon.ico">
+  <link
+  rel="stylesheet"
+  href="http://localhost:8080/style.css"
+  />
+  <script
+    type="module"
+    src="http://localhost:8080/ui.js"
+  ></script>
```

Comment:
I don't think we should be using localhost for these examples - would be nice so they can be deployed standalone on github pages for example - thoughts?

Reply:
Yeah true, it would be cool to serve these (although as far as the user would see, it's exactly the same page being rendered for each framework).

I was using localhost for testing & updated `vite dev` to serve both the components & the demo page.

Can be updated once the lib is released and stable! (and published to GH pages too)